apps/wappush/js/cp_screen_helper.js

```diff
@@ -192,7 +188,8 @@ var CpScreenHelper = (function() {
     // Multimedia Messaging System Enabler). In this case we must not continue.
     if (apns.length === 0) {
       message = finishConfirmDialog.querySelector('strong');
-      message.textContent = _('cp-finish-confirm-dialog-message-no-apns');
+      navigator.mozL10n.setAttributes(message,
+        'cp-finish-confirm-dialog-message-no-apns');
```

Comment:
nit: in the SMS app we adopted a style that reads like this:
```javascript
navigator.mozL10n.setAttributes(
  message, 'cp-finish-confirm-dialog-message-no-apns'
);
```
or
```javascript
navigator.mozL10n.setAttributes(
  message,
  'cp-finish-confirm-dialog-message-no-apns'
);
```
Basically if we need to have a multiline statement, we try to be as symmetrical as possible (symmetrical for the arguments here) to make it easier to read.

Not a blocker though